samples/deployment/aws/build_ami_vftd.yaml

```diff
@@ -0,0 +1,205 @@
+# Create Basic VM For AMI BUILD
+- name: "Build AMI image for vFTD"
+  hosts: localhost
+  remote_user: ubuntu
+  gather_facts: no
+  vars:
+    ftd_build_id: "6.4.0-1304"
+    ec2_instance_type: "c4.xlarge"
+    ec2_base_image: "ami-038f5168"
+    ansible_ssh_private_key_name: "ngfwv-stage-temp-kp.pem"
+    ansible_ssh_private_key_file: "./ngfwv-stage-temp-kp.pem"
+  tasks:
+    - name: "Download QCOW2"
+      get_url:
+        url: "https://...path.../...to.../{{ ftd_build_id }}.qcow2"
+        dest: "./"
+        validate_certs: no
+
+# Prepare Env Section
+    - name: "Create stage key"
+      ec2_key:
+        name: "{{ ansible_ssh_private_key_name }}"
+      register: stage_kp
+    - name: "Save stage key"
+      copy:
+        content: "{{ stage_kp.key.private_key }}"
+        dest: "./ngfwv-stager-temp-kp.pem"
+        mode: 0600
+      when: stage_kp.changed
+    - name: "Create VPC"
+      ec2_vpc_net:
+        name: "Stage VPC"
+        state: present
+        cidr_block: 10.0.0.0/16
+      register: stage_vpc
+    - name: "Create Internet Gateway"
```

Comment:
Lacks an article. "Create **an** Internet Gateway" and the same for other task names.

Reply:
fixed